src/query/query_analytic.cpp

```diff
@@ -782,7 +786,7 @@ qdata_finalize_analytic_func (cubthread::entry *thread_p, ANALYTIC_TYPE *func_p,
   /* set count-star aggregate values */
   if (func_p->function == PT_COUNT_STAR)
     {
-      db_make_int (func_p->value, func_p->curr_cnt);
+      db_make_bigint (func_p->value, func_p->curr_cnt + (int64_t) prm_get_bigint_value (PRM_ID_COUNT_DEBUG));
```

Comment:
How about using 'defined(NDEBUG)'?
```
#if defined(NDEBUG)
db_make_bigint (func_p->value, func_p->curr_cnt);
#else 
db_make_bigint (func_p->value, func_p->curr_cnt + (int64_t) prm_get_bigint_value (PRM_ID_COUNT_DEBUG));
#endif 
```

Reply:
good point.

Reply:
Thank you! I'll change it as you suggest